src/Compression/Enums.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+
+using Foundation;
+using CoreFoundation;
+using ObjCRuntime;
+using CoreVideo;
+
+namespace Compression {
+
+	public enum CompressionAlgorithm {
```

Comment:
are all algorithms available in all versions of the OS ?
if not we need availability attributes on (some of) them

Reply:
I have not checked the headers (and the web docs does not say) but are they all `int` ? and not `NSInteger` or something else ?

Reply:
I'll double check.